apis/Google.Devtools.AspNet/Google.Devtools.AspNet.IntegrationTests/CloudTrace/TraceTest.cs

```diff
@@ -0,0 +1,298 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Google.Cloud.Trace.V1;
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+using Trace = Google.Cloud.Trace.V1.Trace;
+using TraceUtils = Google.Devtools.AspNet.Tests.TraceUtils;
```

Comment:
Doesn't just using Google.Devtools.AspNet.Tests work here? It's fine to create an alias, but I wouldn't expect you to need to.